interfaces/Portal/.env.example

```diff
@@ -24,10 +24,13 @@ NG_TWILIO_ERROR_CODES_URL=https://www.twilio.com/docs/api/errors
 USE_SSO_AZURE_ENTRA=
 
 # Azure Entra environment-specific IDs.
-# Make sure to set these values the same for the 121-service.
+# Make sure to set these two values the same for the 121-service.
 AZURE_ENTRA_CLIENT_ID=
 AZURE_ENTRA_TENANT_ID=
-
+# Set the allowed host names of entra users (comma-separated list, i.e: "example.com,example.net,example.org")
+AZURE_ENTRA_DOMAINS=
+# Azure API URL (should NOT end with a "/"!)
+AZURE_ENTRA_URL=https://login.microsoftonline.com
```

Comment:
These two new ENV-variables should be added/mentioned in the "Unreleased"-section of the CHANGELOG.

Reply:
Done